agent/selfupdater/updater_docker.go

```diff
@@ -112,6 +112,21 @@ func (d *dockerUpdater) CompleteUpdate() error {
 			}
 		}
 
+		// Append /etc/resolv.conf to mount if old container
+		// version is less than v0..0 since /etc/resolv.conf
+		// is required inside the container to update host
+		// networking after boot.
+		v0_7_3, _ := semver.NewVersion("v0.7.3")
+		if v.LessThan(v0_7_3) {
+			parent.info.HostConfig.Mounts = []mount.Mount{
+				{
+					Type:   mount.TypeBind,
+					Source: "/etc/resolv.conf",
+					Target: "/etc/resolv.conf",
+				},
+			}
+		}
+
```

Comment:
@otavio is this what you are referring to in: https://github.com/shellhub-io/shellhub/pull/1126#issuecomment-882846883

Reply:
Yes. The commit log needs to have `agent: ` as a prefix to know which component it is changing.